include/odp/api/spec/ipsec.h

```diff
@@ -914,6 +915,8 @@ typedef struct odp_ipsec_error_t {
 
 			/** Hard lifetime expired: packets */
 			uint32_t hard_exp_packets : 1;
+
+			uint32_t sa_disabled : 1;
```

Comment:
@muvarov Doxygen is missing here but for some reason the Doxygen check didn't detect this?

Reply:
Actually it looks like Doxygen did flag this but the check failed and so Travis gave this a green checkmark:

```
20.12s$ make doxygen-doc 2>&1 |tee doxygen.log
 DXGEN doc/application-api-guide/Doxyfile
/home/travis/build/Linaro/odp/include/odp/api/spec/ipsec.h:919: warning: parameters of member odp_ipsec_error_t::@34::@36::sa_disabled are not (all) documented
 DXGEN doc/helper-guide/Doxyfile
 DXGEN doc/platform-api-guide/Doxyfile
/home/travis/build/Linaro/odp/include/odp/api/spec/ipsec.h:919: warning: parameters of member odp_ipsec_error_t::@34::@36::sa_disabled are not (all) documented
 DXGEN doc/driver-api-guide/Doxyfile
```